src/api/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Good } from '../../../types/Good';
+import './GoodsList.scss';
+
+type Props = {
+  goods: Good[]
+};
+
+export const GoodsList: React.FC<Props> = React.memo(
+  ({ goods }) => (
+
+    <ul className={classNames({ GoodsList: goods.length > 0 })}>
```

Comment:
It seems that you don't need to use `classNames` lib at all. Just use a regular class with constant name.